mne/io/ctf/ctf.py

```diff
@@ -30,7 +30,7 @@ def read_raw_ctf(directory, system_clock='truncate', preload=False,
     Parameters
     ----------
     directory : str
-        Path to the KIT data (ending in ``'.ds'``).
+        Path to the CTF data (ending in ``'.ds'``).
```

Comment:
I am guessing this is also something that was merged in like above. @larsoner, will these pose problems when squashed?